pkgs/development/python-modules/devito/default.nix

```diff
@@ -25,7 +25,7 @@
 
 buildPythonPackage rec {
   pname = "devito";
-  version = "4.8.8";
+  version = "4.8.9";
   format = "setuptools";
```

Comment:
Could you add `pyproject = true` and `build-system = [ setuptools ]`?

Reply:
I have already tried it and it doesn't work unfortunately
```
Checking runtime dependencies for devito-4.8.9-py3-none-any.whl
  + Exception Group Traceback (most recent call last):
  |   File "/nix/store/gdxd1rn8zgjjw08ijghvpazj18n4sjl6-python-runtime-deps-check-hook.py", line 93, in <module>
  |     metadata = get_metadata(args.wheel)
  |                ^^^^^^^^^^^^^^^^^^^^^^^^
  |   File "/nix/store/gdxd1rn8zgjjw08ijghvpazj18n4sjl6-python-runtime-deps-check-hook.py", line 58, in get_metadata
  |     metadata = Metadata.from_raw(raw)
  |                ^^^^^^^^^^^^^^^^^^^^^^
  |   File "/nix/store/mjb4mvqf4rmfam62w3j4094ghadgwvxp-python3.12-packaging-24.1/lib/python3.12/site-packages/packaging/metadata.py", line 701, in from_raw
  |     raise ExceptionGroup("invalid metadata", exceptions)
  | ExceptionGroup: invalid metadata (1 sub-exception)
  +-+---------------- 1 ----------------
    | Traceback (most recent call last):
    |   File "/nix/store/mjb4mvqf4rmfam62w3j4094ghadgwvxp-python3.12-packaging-24.1/lib/python3.12/site-packages/packaging/metadata.py", line 607, in _process_provides_extra
    |     normalized_names.append(utils.canonicalize_name(name, validate=True))
    |                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/nix/store/mjb4mvqf4rmfam62w3j4094ghadgwvxp-python3.12-packaging-24.1/lib/python3.12/site-packages/packaging/utils.py", line 47, in canonicalize_name
    |     raise InvalidName(f"name is invalid: {name!r}")
    | packaging.utils.InvalidName: name is invalid: 'python_version_3.8_'
    | 
    | The above exception was the direct cause of the following exception:
    | 
    | Traceback (most recent call last):
    |   File "/nix/store/mjb4mvqf4rmfam62w3j4094ghadgwvxp-python3.12-packaging-24.1/lib/python3.12/site-packages/packaging/metadata.py", line 696, in from_raw
    |     getattr(ins, key)
    |   File "/nix/store/mjb4mvqf4rmfam62w3j4094ghadgwvxp-python3.12-packaging-24.1/lib/python3.12/site-packages/packaging/metadata.py", line 504, in __get__
    |     value = converter(value)
    |             ^^^^^^^^^^^^^^^^
    |   File "/nix/store/mjb4mvqf4rmfam62w3j4094ghadgwvxp-python3.12-packaging-24.1/lib/python3.12/site-packages/packaging/metadata.py", line 609, in _process_provides_extra
    |     raise self._invalid_metadata(
    | packaging.metadata.InvalidMetadata: 'python_version_3.8_' is invalid for 'provides-extra'
    +------------------------------------
```

Reply:
Oh, sorry. I think we had the same conversation in the previous update.